sphinx_immaterial/__init__.py

```diff
@@ -47,11 +48,11 @@
 
 def _get_html_translator(
     base_translator: Type[sphinx.writers.html5.HTML5Translator],
-) -> Type[sphinx.writers.html5.HTML5Translator]:
+) -> sphinx.writers.html5.HTML5Translator:
```

Comment:
I think the old type was correct here.

Reply:
I wasn't aware of the significance of `Type` (seemed redundant to me).